docs/concepts/workloads/controllers/deployment.md

```diff
@@ -50,7 +54,17 @@ NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
 nginx-deployment   3         0         0            0           1s
 ```
 
-This indicates that the Deployment's number of desired replicas is 3 (according to deployment's `.spec.replicas`), the number of current replicas (`.status.replicas`) is 0, the number of up-to-date replicas (`.status.updatedReplicas`) is 0, and the number of available replicas (`.status.availableReplicas`) is also 0.
+This indicates that the Deployment's number of desired replicas is 3 (according to deployment's `.spec.replicas`),
+the number of current replicas (`.status.replicas`) is 0, the number of up-to-date replicas (`.status.updatedReplicas`)
+is 0, and the number of available replicas (`.status.availableReplicas`) is also 0.
```

Comment:
fine with follow up, maybe it would be worth explaining some basic terminology about deployments at some point in one place (minReady/maxSurge/etc..)